cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
It is if the test program doesn’t write to stdout ;)

which, in my opinion, is how most tests should behave. I really don’t like how the Ceylon typechecker or backend tests completely flood my terminal with information that’s useless 99% of the time.

What default file name would you suggest? `reports/tap`? But then I think it’s inconsistent that the HTML option is called just `--report` instead of `--html`.